pkg/ui/workspaces/cluster-ui/src/jobs/util/duration.tsx

```diff
@@ -38,12 +38,18 @@ export class Duration extends React.PureComponent<{
       const fractionCompleted = job.fraction_completed;
       if (!startedAt || !modifiedAt || fractionCompleted === 0) {
         return null;
+      } else if (fractionCompleted < 0.05) {
+        return <span className={className}>Initializing...</span>;
       }
       const duration = modifiedAt.diff(startedAt);
-      const remaining = duration / fractionCompleted - duration;
+      const remaining = moment.duration(
+        duration / fractionCompleted - duration,
+      );
       return (
         <span className={className}>
-          {formatDuration(moment.duration(remaining)) + " remaining"}
+          {remaining >= moment.duration(1, "minutes")
+            ? formatDuration(remaining)
+            : "Less than a minute" + " remaining"}
```

Comment:
Looks like `"Less than a minute" + " remaining"` should just be `"Less than a minute remaining"` since there isn't any value being inserted.

Reply:
Ah the + is for the case where there is more than a minute remaining -- it does look weird though as is, so I updated it to a template string to make it a bit more clear.